.github/workflows/release-packages.yaml

```diff
@@ -43,3 +43,43 @@ jobs:
           asset_path: ${{ steps.create_packages.outputs.deb_package }}
           asset_name: ${{ steps.create_packages.outputs.deb_package_name }}
           asset_content_type: application/x-deb
+
+  windows-msi-package:
+  env:
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Fetch dependencies
+      run: |
+        choco install winflexbison3
+    - uses: microsoft/setup-msbuild
```

Comment:
Does it make sense to fix the version in this script as well?

Reply:
yes, this bit still needs fixing